plugins/src/signaller/imp.rs

```diff
@@ -347,13 +357,22 @@ impl ObjectSubclass for Signaller {
 impl ObjectImpl for Signaller {
     fn properties() -> &'static [glib::ParamSpec] {
         static PROPERTIES: Lazy<Vec<glib::ParamSpec>> = Lazy::new(|| {
-            vec![glib::ParamSpecString::new(
-                "address",
-                "Address",
-                "Address of the signalling server",
-                Some("ws://127.0.0.1:8443"),
-                glib::ParamFlags::READWRITE,
-            )]
+            vec![
+                glib::ParamSpecString::new(
+                    "address",
+                    "Address",
+                    "Address of the signalling server",
+                    Some("ws://127.0.0.1:8443"),
+                    glib::ParamFlags::READWRITE,
+                ),
+                glib::ParamSpecString::new(
+                    "cafile",
+                    "cafile",
+                    "cafile",
```

Comment:
Path to a Certificate file to add to the set of roots the TLS connector will trust